source/fe/fe_q_base.cc

```diff
@@ -605,13 +605,38 @@ FE_Q_Base<PolynomialType, dim, spacedim>::get_face_interpolation_matrix(
 }
 
 
-
 template <class PolynomialType, int dim, int spacedim>
 void
 FE_Q_Base<PolynomialType, dim, spacedim>::get_subface_interpolation_matrix(
   const FiniteElement<dim, spacedim> &x_source_fe,
   const unsigned int                  subface,
   FullMatrix<double> &                interpolation_matrix) const
+{
+  get_subface_interpolation_matrix_implemetation(x_source_fe,
+                                                 subface,
+                                                 interpolation_matrix);
+}
+
+template <class PolynomialType, int dim, int spacedim>
+void
+FE_Q_Base<PolynomialType, dim, spacedim>::get_subface_interpolation_matrix(
+  const FiniteElement<dim, spacedim> &x_source_fe,
+  const unsigned int                  subface,
+  FullMatrix<std::complex<double>> &  interpolation_matrix) const
+{
+  get_subface_interpolation_matrix_implemetation(x_source_fe,
+                                                 subface,
+                                                 interpolation_matrix);
```

Comment:
`get_subface_interpolation_matrix()` calls the private template implementation. If this is good I can do it for the rest of the elements in deal.II. Otherwise, let me know if you have a better suggestion.